instrumentation-core/spring/spring-web-3.1/src/main/java/io/opentelemetry/instrumentation/spring/httpclients/RestTemplateInterceptor.java

```diff
@@ -33,26 +32,19 @@
 
   private final Tracer tracer;
 
-  public RestTemplateInterceptor(Tracer tracer) {
+  public RestTemplateInterceptor(final Tracer tracer) {
```

Comment:
Think we're not supposed to have this per #921 (though we can delete at the same time so no worries)